src/FNISPatches.py

```diff
@@ -88,7 +88,7 @@ def description(self):
         return self.__tr("Configures the patches which FNIS applies to the game.")
 
     def version(self):
-        return mobase.VersionInfo(1, 0, 0, mobase.ReleaseType.prealpha)
+        return mobase.VersionInfo(1, 0, 1, 0)
```

Comment:
The last argument's still supposed to be an enum, isn't it? `0` is still pre-alpha, so it's not making the text in the GUI any better if it's zero.

Reply:
There is a 4-int overload, not sure if it works properly though since enums are classes that extends `int` for Boost.Python.

Reply:
It works.  I can't tell you why it works but it does.  The version in the plugins settings goes from "1.0.0.0 pre-alpha" to "1.0.1.0".  /shrug

Reply:
Does Final make it show up as `1.0.1 Final` or just `1.0.1`?

Reply:
`1.0.1.0`.  So I guess putting final in there would work or we could just reclaim the 4th build number as otherwise it always shows 0.

Reply:
Final is the default release type, so `mobase.VersionInfo(1, 0, 1, 0)` is 1.0.1.0 Final and `mobase.VersionInfo(1, 0, 1)` is 1.0.1 Final, but Final is not shown.

Reply:
Changed to final for this since all the other plugins do that.  It kind of sounds like the most appropriate thing would be to update the plugins settings page to only display the first 3 versions and then a text flag if needed.  Maybe.  Not like it really matters.

Reply:
Not that it really matters but you should use the uppercase version `FINAL` or omit the argument.